### PR TITLE
Feature/harmony publish template

### PR DESCRIPTION
**Pull request by tokejepsen**
resolves https://github.com/pypeclub/fourthwall-config/issues/3

### CODE REVIEW

File: pype/plugins/global/publish/integrate_new.py [hunk 1]
```diff
@@ -81,7 +81,8 @@ class IntegrateAssetNew(pyblish.api.InstancePlugin):
                 "assembly",
                 "fbx",
                 "textures",
-                "action"
+                "action",
+                "template"
```
Comment (mkolar) [inline pype/plugins/global/publish/integrate_new.py:85]: I think we should be more specific with this family. Template sounds too generic, considering it's only harmony compatible family. How about `harmony.template` or something specific like that

Comment (tokejepsen) [reply pype/plugins/global/publish/integrate_new.py:85]: Sure, `harmony.template` sounds good.
Was looking at `nukenodes` but like the dot notation better.

Comment (tokejepsen) [reply pype/plugins/global/publish/integrate_new.py]: Done

**Comment by tokejepsen** [comment]
I dont have merge rights, if thats what you wanted me to do? @mkolar

**Comment by mkolar** [comment]
my bad. merging now.

**Pull request merged**